### PR TITLE
[Doc] Update type cast

### PR DESCRIPTION
**Pull request by neozhaoliang**

### CODE REVIEW

File: docs/lang/articles/type_system/type.md [hunk 1]
```diff
@@ -79,14 +79,16 @@ In the Taichi scope, the type of a variable is *statically typed* upon initializ
...
+- You can also use primitive types like `ti.f32`, `ti.i64` to convert a variable of scalar type to other types:
 
   ```python
   @ti.kernel
   def foo():
       a = 3.14
-      b = int(a)    # 3
-      c = float(b)  # 3.0
+      x = int(a)    # 3
+      y = float(b)  # 3.0
```
Comment (strongoier) [inline docs/lang/articles/type_system/type.md:89]: What is `b` here?

File: docs/lang/articles/type_system/type.md [hunk 2]
```diff
@@ -79,14 +79,16 @@ In the Taichi scope, the type of a variable is *statically typed* upon initializ
...
+      x1 = ti.i32(x)  # 3
+      y1 = ti.f64(x)  # 3.14
```
Comment (strongoier) [inline docs/lang/articles/type_system/type.md:91]: Converting from `x` will not get 3.14.

Comment (neozhaoliang) [reply docs/lang/articles/type_system/type.md]: Sorry my mistake, it should be fixed now.

File: docs/lang/articles/type_system/type.md [hunk 3]
```diff
@@ -79,14 +79,16 @@ In the Taichi scope, the type of a variable is *statically typed* upon initializ
...
+- You can also use primitive types like `ti.f32`, `ti.i64` to convert a variable of scalar type to other scalar types:
 
   ```python
   @ti.kernel
   def foo():
       a = 3.14
-      b = int(a)    # 3
-      c = float(b)  # 3.0
+      x = int(a)    # 3
+      y = float(a)  # 3.0
```
Comment (strongoier) [inline docs/lang/articles/type_system/type.md:89]: 3.0 is incorrect.

File: docs/lang/articles/type_system/type.md [hunk 4]
```diff
@@ -79,14 +79,16 @@ In the Taichi scope, the type of a variable is *statically typed* upon initializ
       c = ti.cast(b, ti.f32)  # 3.0
   ```
 
-- You can also use Python's builtin `int()` or `float()` to convert a value to the default integer type or to the default floating-point type:
+- You can also use primitive types like `ti.f32`, `ti.i64` to convert a variable of scalar type to other scalar types:
```
Comment (Vissidarte-Herman) [inline docs/lang/articles/type_system/type.md:82]:
```suggestion
- As of v1.1.0, you are allowed to use primitive types such as `ti.f32` and `ti.i64` to convert a scalar variable to a different scalar type:
```

**Review by Vissidarte-Herman — CHANGES_REQUESTED** [review]
Minor editorial udpates. Otherwise, LGTM!

**Review by Vissidarte-Herman — APPROVED** [review]
LGTM!

**Pull request merged**